blender/arm/material/cycles_nodes/nodes_shader.py

```diff
@@ -170,10 +172,14 @@ def parse_holdout(node: bpy.types.ShaderNodeHoldout, out_socket: NodeSocket, sta
         state.out_occlusion = '0.0'
 
 
-def parse_bsdfrefraction(node: bpy.types.ShaderNodeBsdfRefraction, out_socket: NodeSocket, state: ParserState) -> None:
-    # c.write_normal(node.inputs[3])
-    pass
-
+def parse_bsdfrefraction(node: bpy.types.ShaderNodeBsdfRefraction, out_socket: NodeSocket, state: ParserState) -> None: 
+    if state.parse_surface:
+        state.out_basecol = c.parse_vector_input(node.inputs[0])
+        c.write_normal(node.inputs[3])
+        state.out_roughness = c.parse_value_input(node.inputs[1])
+    if state.parse_opacity:
+        state.out_opacity = '0.0'
+        state.out_rior = c.parse_value_input(node.inputs[2])
```

Comment:
There's this implementation for the Refraction BSDF node, but when using this, Armory prints `WARNING: Node tree "Refractive": material node type BSDF_REFRACTION not supported` to the console. The node is missing an entry in [node_meta.py](https://github.com/armory3d/armory/blob/0d2a0d5c31994e15c1ff52d4d10ea73fe571491d/blender/arm/material/node_meta.py#L65).

Reply:
Done !